text/0876-support-for-es-accessor-decorators.md

```diff
@@ -0,0 +1,420 @@
+---
+stage: accepted
+start-date: 2022-11-27T00:00:00.000Z # In format YYYY-MM-DDT00:00:00.000Z
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams: # delete teams that aren't relevant
+  - framework
+  - learning
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/876
+project-link:
+suite: 
+---
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+# Support for ES (`accessor`) Decorators
+
+## Summary
+
+Introduce support for the [ECMAScript Decorators Proposal][spec] for the key decorators which are part of Ember’s modern programming model, `@service` and `@tracked`:
+
+```js
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking/accessor';
+import { service } from '@ember/service/accessor';
```

Comment:
will there be a future RFC to simplify imports?
we have _a lot_ of different import paths now, and it seems this is rapidly growing over time